app/components/spin-button.js

```diff
@@ -106,7 +107,9 @@ export default Ember.Component.extend({
       this._spinner = createSpinner( element );
       this._spinner.spin(element.querySelector('.spin-button-spinner'));
     }
+  },
 
+  createDisabledTimeout: function() {
```

Comment:
I was not sure what `createDisabledTimeout` does, so I looked at the body of the method, and it seems to be doing a little bit more than that, since it also cancels a running timeout. So maybe `reset` instead of `create`.

Also, `disabled` got me be confused for a bit... Do we create a timeout that is somehow disabled by default, until we start it manually or something? (I don't think that's even possible, but that is what crossed my mind reviewing this). And anyway, in the context of this component, I think just `timeout` is obvious enough and does not need any extra qualification (simple is better if it is already expressive enough).

Finally, they use `_timer` internally, so instead of `timeout`, maybe the method name could use that for consistency? Not sure about this last one though... So I suggest simply:

```
resetTimer

# or 

resetTimeout
```

Hmm... The second one feels better. `timeout` is quite clear, while `timer` is ambiguous. What are we timing?

But the original author decided to call his var `_timer`, so `resetTimer` is more consistent with the surrounding code...

In the context of a PR to a component, maybe consistency with the existing code is more important, so shall we go with `resetTimer`?